.travis.yml

```diff
@@ -16,3 +31,5 @@ branches:
   only:
   - master
   - /\d{1,3}\.\d{1,3}\.\d{1,3}/
+env:
+  - CYPRESS_baseUrl=https://discovery.dev.internal.smartcolumbusos.com
```

Comment:
This should be paramatized. It shouldn't point to a specific environment or be columbus specific here. That all should get passed in by the job runner.

Reply:
Yeah this was wrong. These test are for local tests so baseUrl should still be "http://locahost:9001". However, I did set an environment variable in travis that overrides window.API_HOST to the real backend.